lib/passkit.rb

```diff
@@ -18,36 +18,70 @@ class << self
   def self.configure
     self.configuration ||= Configuration.new
     yield(configuration) if block_given?
+    configuration.validate!
   end
 
   class Configuration
     attr_accessor :available_passes,
       :web_service_host,
-      :certificate_key,
-      :private_p12_certificate,
-      :apple_intermediate_certificate,
+      :p12_password,
+      :p12_certificate,
+      :p12_key,
+      :wwdc_cert,
       :apple_team_identifier,
-      :pass_type_identifier
+      :pass_type_identifier,
+      :format_version,
+      :dashboard_username,
+      :dashboard_password
 
-    DEFAULT_AUTHENTICATION = proc do
-      authenticate_or_request_with_http_basic("Passkit Dashboard. Login required") do |username, password|
-        username == ENV["PASSKIT_DASHBOARD_USERNAME"] && password == ENV["PASSKIT_DASHBOARD_PASSWORD"]
```

Comment:
We are loosing the default taken from the ENV variable with this change. Can you please set both dashboard_username and dashboard_password to the ENV variables as default?